City-of-Helsinki/events-helsinki-monorepo
# LIIKUNTA-582, LIIKUNTA-585 | feat: add support for hierarchical menus & universal bar menu in header

## Description

### feat: add support for hierarchical menus & universal bar menu in header

refs LIIKUNTA-582 (add support for hierarchical menus in header)
refs LIIKUNTA-585 (add support for universal bar menu in header)

## Issues

### Closes

[LIIKUNTA-582](https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-582)
[LIIKUNTA-585](https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-585)

### Related

## Testing

 - Check out PR's branch locally
 - `yarn`
 - `cd apps/sports-helsinki`
   - set local variables in `.env.local`
     - NEXT_PUBLIC_CMS_HEADER_UNIVERSAL_BAR_MENU_NAME_FI="Sports Helsinki Universal bar FI"
     - NEXT_PUBLIC_CMS_HEADER_UNIVERSAL_BAR_MENU_NAME_EN="Sports Helsinki Universal bar EN"
     - NEXT_PUBLIC_CMS_HEADER_UNIVERSAL_BAR_MENU_NAME_SV="Sports Helsinki Universal bar SV"
   - `yarn dev`

### Automated tests

### Manual testing

## Screenshots

![image](https://github.com/City-of-Helsinki/events-helsinki-monorepo/assets/77663720/e71adce7-c37e-4ce0-b760-75afc487fff2)

![image](https://github.com/City-of-Helsinki/events-helsinki-monorepo/assets/77663720/8fe0567c-357c-46c0-9308-67f0a65b2c91)

![image](https://github.com/City-of-Helsinki/events-helsinki-monorepo/assets/77663720/5d46c287-1cd4-4216-bc1d-b7bbcce214af)

![image](https://github.com/City-of-Helsinki/events-helsinki-monorepo/assets/77663720/0083ce16-2c16-4f8b-b240-4a31de9b9551)

![image](https://github.com/City-of-Helsinki/events-helsinki-monorepo/assets/77663720/d70cd1a0-17f2-49f1-abf6-b3dcd2dcf435)

![image](https://github.com/City-of-Helsinki/events-helsinki-monorepo/assets/77663720/10b9192a-a425-4573-b3a4-3f9c816e947a)

![image](https://github.com/City-of-Helsinki/events-helsinki-monorepo/assets/77663720/17c525bf-3548-4492-9c97-d746495a0bfd)

![image](https://github.com/City-of-Helsinki/events-helsinki-monorepo/assets/77663720/ae784fff-30f6-4986-aef8-840e7eb827d8)

## Additional notes


[LIIKUNTA-582]: https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-582?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LIIKUNTA-585]: https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-585?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
